src/components/AuthForm.tsx

```diff
@@ -173,6 +173,19 @@ export function AuthForm({ state }: IAuthFormProps) {
     })
 
     processResponse({ session, error, stateType: 'register' })
+
+    if(!error){
+      showAlert({
+        title: "Registrasi Berhasil!",
+        message: "Terima kasih telah mendaftar. Silahkan melakukan verifikasi dengan mengklik tautan yang kami kirimkan melalui email."
+      })
+    }
+    else if(!error){
```

Comment:
kok else if nya sama?

Gini aja cukup gak ya?

```suggestion
    else {
```

Reply:
Oh iya mas bener, bisa juga